lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -1091,57 +1131,60 @@ sub job_schedule_iso {
         }
     }
 
-    my @ids;
-
     # the jobs are now sorted parents first
-    # remember ids of created parents and pass them to _START_AFTER_JOBS/_PARALLEL_JOBS of children
-    my %testsuite_ids;    # key: "suite:machine", value: array of job ids
-
-    for my $settings (@{$jobs || []}) {
-        my $prio     = delete $settings->{PRIO};
-        my $group_id = delete $settings->{GROUP_ID};
-
-        # convert testsuite names in START_AFTER_TEST/PARALLEL_WITH to job ids
-        for my $after (_parse_dep_variable($settings->{START_AFTER_TEST}, $settings)) {
-            if (defined $testsuite_ids{$after}) {
-                $settings->{_START_AFTER_JOBS} //= [];
-                push @{$settings->{_START_AFTER_JOBS}}, @{$testsuite_ids{$after}};
-            }
-            else {
-                warn "START_AFTER_TEST=" . $after . " not found, maybe a typo or a dependency cycle";
-            }
-        }
-        for my $after (_parse_dep_variable($settings->{PARALLEL_WITH}, $settings)) {
-            if (defined $testsuite_ids{$after}) {
-                $settings->{_PARALLEL_JOBS} //= [];
-                push @{$settings->{_PARALLEL_JOBS}}, @{$testsuite_ids{$after}};
+
+    my @ids     = ();
+    my $coderef = sub {
+        my @jobs = ();
+        # remember ids of created parents
+        my %testsuite_ids;            # key: "suite:machine", value: array of job ids
+
+        for my $settings (@{$jobs || []}) {
+            my $prio     = delete $settings->{PRIO};
+            my $group_id = delete $settings->{GROUP_ID};
+
+            # create a new job with these parameters and count if successful, do not send job notifies yet
+            my $job;
+            try {
```

Comment:
I'm wondering if this is good idea. To call try in DBIx txn_do if txn_do rely on exceptions. Probably not very good.